src/Handlers/SystemWeb/HttpHandlerEndpointConventionBuilder.cs

```diff
@@ -102,15 +108,23 @@ public void Add(Action<EndpointBuilder> convention)
 
     public override IChangeToken GetChangeToken() => new CompositeChangeToken(_managers.Select(m => m.GetChangeToken()).ToArray());
 
-    private static Task DefaultHandler(HttpContextCore context)
+    private static RequestDelegate BuildDefaultHandler(IServiceProvider services)
     {
-        if (context.Features.GetRequiredFeature<IHttpHandlerFeature>().Current is { } handler)
+        var builder = new ApplicationBuilder(services);
+
```

Comment:
nit:- empty line

Reply:
that new line is actually a style I prefer - I create the builder, a break, then the builder is added to, then a new line, then the build. It helps separate out what middleware you're adding to the builder.